base/multidimensional.jl

```diff
@@ -705,15 +704,15 @@ cumprod(A::AbstractArray, axis::Integer=1) = accumulate(*, A, axis)
 """
     cumprod!(B, A, dim::Integer=1)
 
-Cumulative product of `A` along a dimension, storing the result in `B`. The dimension defaults to 1.
+Cumulative product of `A` along a dimension, storing the result in `B`. 
```

Comment:
Woops, turns out the CI failures are legitimate and related to the trailing space here.